src/controller/editingcontroller.js

```diff
@@ -61,6 +64,19 @@ export default class EditingController {
 		this.mapper = new Mapper();
 
 		/**
+		 * Defines whether controller is in read-only mode.
```

Comment:
Yea, I was thinking about it too, we even talked about it with @oskarwrobel. On the one hand, we can say that editing controller is a facade, so it can have this property. On the other, we don't use it this way usually. In most cases, we use nested properties directly. So I agree that we could not make an exception here.